sdk/core/Azure.Core.Experimental/src/Serialization/ISerializedMemberNameProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Reflection;
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// Provides mappings for member names to serialized property names.
+    /// </summary>
+    public interface ISerializedMemberNameProvider
+    {
+        /// <summary>
+        /// Maps a member <see cref="MemberInfo.Name"/> to a serialized property name.
```

Comment:
Nit - It uses the member name, attributes, etc., so should we just say `Maps a <see cref="MemberInfo"/> to a serialized name.`?